.env-sample

```diff
@@ -0,0 +1,15 @@
+PORT=3000
+DB_NAME=my_database
+DB_USERNAME=postgres
+DB_PASSWORD=secret
+DB_HOST=localhost
+DB_PORT=5432
+
+
+TEST_DB_NAME=my_testdb
+TEST_DB_USERNAME=testuser
+TEST_DB_PASSWORD=testpassword
+TEST_DB_HOST=localhost
+TEST_DB_PORT=5432
+
+
```

Comment:
To be consistent with what you have in .config.js file, can you please edit the file to look something like 
DATABASE_URL_TEST=postgres://dbuser:pass@host:post/dbname